### ISSUE TITLE
AttributeError: 'Pyfhel.PyCtxt.PyCtxt' object has no attribute 'copy'

### ISSUE THREAD

**Issue by boxer-code** [ibarrond/Pyfhel#164]
**Description**
While trying to execute the Demo_5_CS_Client.py example the script fails with: "AttributeError: 'Pyfhel.PyCtxt.PyCtxt' object has no attribute 'copy'". Am I missing something about the setup? I've already checked that I'm using the latest versions.

**Code To Reproduce Error**
I'm using the example code of  Demo_5_CS_Client.py.

**Setup:**
 - OS: Ubuntu 22.04 (WSL)
 - Python: 3.10.6
 - C compiler version: GCC 11.3.0
 - Pyfhel Version: Pyfhel 3.3.1

Full logs:
````
[Client] Initializing Pyfhel session and data...
[Client] sending HE_client=<ckks Pyfhel obj at 0x7fc736d464c0, [pk:Y, sk:Y, rtk:Y, rlk:Y, contx(n=8192, t=0, sec=128, qi=[30, 30, 30, 30, 30], scale=1073741824.0, )]> and cx=<Pyfhel Ciphertext at 0x7fc736d467f0, scheme=ckks, size=2/2, scale_bits=30, mod_level=0>
[Client] launching server (could be launched separately)...
[Server] mock started!...
[Client] server initialized...
[Server] received HE_server=<ckks Pyfhel obj at 0x7fc735699480, [pk:Y, sk:-, rtk:Y, rlk:Y, contx(n=8192, t=0, sec=128, qi=[30, 30, 30, 30, 30], scale=1073741824.0, )]> and cx=<Pyfhel Ciphertext at 0x7fc7356993a0, scheme=ckks, size=2/2, scale_bits=30, mod_level=0>
[Server] Average computed! Responding: c_mean=<Pyfhel Ciphertext at 0x7fc736d467f0, scheme=ckks, size=2/2, scale_bits=60, mod_level=2>
Traceback (most recent call last):
  File "/home/helen/Demo_5_CS_Client.py", line 103, in <module>
    c_res = c_mean.copy() # Copying with a single command
AttributeError: 'Pyfhel.PyCtxt.PyCtxt' object has no attribute 'copy'
````

**Comment by ibarrond** [comment]
Solved! installing `Pyfhel==3.3.2` should do the trick.

**Comment by boxer-code** [comment]
Thanks! I can confirm that it works now.
